python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -499,14 +460,22 @@ async def enable_devices(self):
         self.log.info("Enable devices")
         self.disable_task.cancel()
         try:
-            enable_commands = [
+            for reset_command in [
+                commands.SafetyReset(what="25"),  # Reset e-stops.
                 # commands.TopEndChillerResetAlarm(),
                 # commands.MainPowerSupplyResetAlarm(),
                 # commands.MirrorCoverLocksResetAlarm(),
                 # commands.MirrorCoversResetAlarm(),
                 # commands.AzimuthAxisResetAlarm(),
                 # commands.ElevationAxisResetAlarm(),
                 commands.CameraCableWrapResetAlarm(),
+            ]:
+                try:
+                    await self.send_command(reset_command, do_lock=False)
+                except Exception as e:
+                    self.log.warning(f"Command {reset_command} failed; continuing: {e}")
```

Comment:
Logging the exception like this give little information about the traceback. Consider using

```
self.log.exception(f"Command {reset_command} failed; continuing...")
```

Reply:
This was intentional as I don't believe the traceback provides any useful information. But I should use `e!r` instead of `e`.